.github/workflows/update-checksums.yml

```diff
@@ -43,17 +45,52 @@ jobs:
           service_url: '${{ vars.TOKEN_MINTER_SERVICE_URL }}'
           requested_permissions: '{"repositories":["secure-setup-terraform"],"permissions":{"pull_requests":"write","contents":"write"}}'
       # Create a pull request for review
+      # Use the GitHub API to ensure commits are signed
       - id: 'create-pull-request'
         if: '${{ env.CHANGES }}'
-        uses: 'peter-evans/create-pull-request@b4d51739f96fca8047ad065eccef63442d8e99f7' # ratchet:peter-evans/create-pull-request@v4
+        uses: 'actions/github-script@98814c53be79b1d30f795b907e553d8679345975' # ratchet:actions/github-script@v6
+        env:
+          DEFAULT_BRANCH: '${{ github.event.repository.default_branch }}'
         with:
-          token: '${{ steps.mint-token.outputs.token }}'
-          add-paths: 'terraform-checksums.json'
-          commit-message: 'chore: [automated] checksum updates'
-          committer: 'Token Minter <abcxyz-token-minter-github-app@google.com>'
-          author: 'Token Minter <abcxyz-token-minter-github-app@google.com>'
-          delete-branch: true
-          branch: '${{ env.PR_BRANCH }}'
-          title: 'chore: Terraform checksum updates for ${{ env.UPDATE_DATE }}'
-          body: |-
-            Adds Terraform binary checksums for ${{ env.CHANGES }} versions: ${{ env.VERSIONS }}
+          github-token: '${{ steps.mint-token.outputs.token }}'
+          retries: '3'
+          script: |-
+            const fs = require("fs");
+
+            const checksumFilePath = `${ process.env.GITHUB_WORKSPACE }/terraform-checksums.json`
```

Comment:
```suggestion
            const checksumFilePath = `${{ github.workspace }}/terraform-checksums.json`
```

Reply:
I dont like using the interpolation syntax with the github interpolation because it cant be formatted. I've decided to set top level vars and use those instead which allows me to format outside the workflow and paste paste back in.